internal/gcsx/inactive_timeout_reader.go

```diff
@@ -236,7 +236,7 @@ func (itr *InactiveTimeoutReader) closeGCSReader() {
 	}
 
 	// Not printing the timeout explicitly, as can be refer from the code/config.
-	logger.Infof("Closing reader for object %q due to inactivity.\n", itr.object.Name)
+	logger.Debugf("Closing reader for object %q due to inactivity.\n", itr.object.Name)
```

Comment:
```suggestion
	logger.Tracef("Closing reader for object %q due to inactivity.\n", itr.object.Name)
```

I think, currently all the GCSFuse logs are at trace level.

Reply:
My apologies! This is what happens when you ask Gemini to fix a bug for you.